applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -45,10 +45,14 @@ <h3 class="signup-confirmation--success">Check your Inbox!</h3>
                     </div>
 
                     <form action="@LinkTo("/email")" method="post" target="magicframe" class="newsletter-card__signup">
+                        <input class="newsletter-card__text-input u-h"
+                          type="text"
+                          name="name"
+                          placeholder="Email address"/>
                         <input class="newsletter-card__text-input js-newsletter-card__text-input"
                           type="email"
                           name="email"
-                          placeholder="Email address" required />
```

Comment:
fixed in `d08204f02cde999a0e8eab145524dff130519854`

Reply:
this was changed to distract the bot (so there's not 1 required field 1 optional field) but we can try making it required again. it's not a big deal as there's javascript validating this field too and, obviously, the worst case scenario is that the form gets rejected server side because the email is empty